include/model.php

```diff
@@ -121,6 +121,9 @@ public function __construct( Options &$options ) {
 
 		// Just in case someone would like to display the language description ;).
 		add_filter( 'language_description', '__return_empty_string' );
+
+		add_action( 'delete_transient_pll_languages_list', array( $this->languages, 'delete_transient_from_options_table' ) );
+		add_action( 'set_transient_pll_languages_list', array( $this->languages, 'set_transient_to_options_table' ) );
```

Comment:
Since you are using the constant in `Languages`, it would be better to use it here the action name too.

Reply:
Done in a0e53137.